sdk/communication/Azure.Communication.CallAutomation/src/Models/DtmfTone.cs

```diff
@@ -13,5 +13,75 @@ namespace Azure.Communication.CallAutomation
     [JsonConverter(typeof(EquatableEnumJsonConverter<DtmfTone>))]
     public readonly partial struct DtmfTone
     {
+        /// <summary>
+        /// The Dtmf Tone in character format.
+        /// </summary>
+        public string ToChar()
```

Comment:
I find confusing that the name is `ToChar` but it returns a `string`. 
Shouldn't it return a `char` instead?

Reply:
changed to char